### DISCUSSION TITLE
Log as error message in the cypress command log

### DISCUSSION THREAD

**Discussion by RVledder** [cypress-io/cypress#24615] [Questions and Help] [answered]
With [log][1] it is possible to log a message to the _cypress command log_. This has a blue color and is for information.

Is it also possible to log an error message?

The use-case is this: I have added a [command][2]. But for this command to successfully execute the environment variables have to be set. I would like the task to pause when no environment variable has been set, and the refer to the docs:

```typescript
Cypress.Commands.add('login', () => {

  if (!Cypress.env('SOME_API_URL')) {
    // ideally this would be a red message instead of blue
    cy.log('Can\'t login because not all necessary Cypress environment variables have been set.');
    cy.log('Please check the readme in this project.');
    cy.pause();
  }

  cy.request({
    url: Cypress.env('SOME_API_URL'),
    // left out for brevity
  }).then(resp => {
    // left out for brevity
    cy.log('succesfully logged in.');
  }).as('login');
}
```

  [1]: https://docs.cypress.io/api/commands/log
  [2]: https://docs.cypress.io/api/cypress-api/custom-commands

**Comment by emilyrohrbough** [comment, marked as answer]
@StudioSpindle Just curious, but why not just throw an error? You'll need to make updates and restart the test to successfully run after the envs have been configured.

**Reply by RVledder** [reply]
Ahh, that is a good suggestion. Didn't think of that.

Tried it and that looks perfect. Ty
